doc/source/docs/user_guide/mapping.rst

```diff
@@ -268,12 +268,12 @@ You can also create these other plots using the ``GeoDataFrame.plot.<kind>`` acc
     @savefig pandas_bar_plot.png
     gdf.plot.bar()
 
-For more information check out the `pandas documentation <https://pandas.pydata.org/pandas-docs/stable/user_guide/visualization.html>`_.
+For more information, see `Chart visualization <https://pandas.pydata.org/pandas-docs/stable/user_guide/visualization.html>`_.
```

Comment:
maybe we can keep something " .. in the pandas documentation" (after the link, not in the link text), to keep it clear this is referring to the main pandas docs and not another page in our docs?